testing/skia_gold_client/lib/skia_gold_client.dart

```diff
@@ -257,11 +257,15 @@ class SkiaGoldClient {
     final ProcessResult result = await _runCommand(imgtestCommand);
 
     if (result.exitCode != 0) {
-      // We do not want to throw for non-zero exit codes here, as an intentional
-      // change or new golden file test expect non-zero exit codes. Logging here
-      // is meant to inform when an unexpected result occurs.
-      print('goldctl imgtest add stdout: ${result.stdout}');
-      print('goldctl imgtest add stderr: ${result.stderr}');
+      final StringBuffer buf = StringBuffer()
+        ..writeln('Skia Gold imgtest add failed.')
```

Comment:
Gonna nit this error message to try to encourage consistency across flutter/flutter and flutter/engine. In the framework we provide links to the dashboard for triage etc.

Reply:
This is where we handle this case in the framework: https://github.com/flutter/flutter/blob/b09a015ff5677da6cd9bb631ca44b801953cb2ad/packages/flutter_goldens/lib/skia_client.dart#L237

Reply:
So here it would be (I think the result-state.json file is not necessary):

```dart
final StringBuffer buf = StringBuffer()
        ..writeln('Skia Gold received an unapproved image in post-submit ')
        ..writeln('testing. Golden file images in flutter/engine are triaged ')
        ..writeln('in pre-submit during code review for the given PR.')
        ..writeln()
        ..writeln('Visit https://flutter-engine-gold.skia.org/ to view and approve ')
        ..writeln('the image(s), or revert the associated change. For more ')
        ..writeln('information, visit the wiki: ')
        ..writeln('https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter')
        ..writeln()
        ..writeln('Debug information for Gold --------------------------------')
        ..writeln('stdout: ${result.stdout}')
        ..writeln('stderr: ${result.stderr}');
      throw SkiaException(buf.toString());
```

Reply:
Nit appreciated and updated.